### ISSUE TITLE
A11y: Add text size selection

### ISSUE THREAD

**Issue by jhermann** [forem/forem#6764]
**Is your feature request related to a problem? Please describe.**

While I can change the *font* used for text display, I cannot change the *size* of text. While somewhat manageable with `Ctrl +` (zoom) in desktop situations, this typically fails on mobile devices, especially when using the app.

**Describe the solution you'd like**

Add text size selection (e.g. via size buttons "S M L XL" or "A- A A+ A++", with M/A being the default) to the global settings.

Ideally also to the top of post displays, with only *temporarily* changing the size for this one page view. Here two buttons (`A▿` and `A▵`) would be sufficient.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by ludwiczakpawel** [comment]
Hey @jhermann !

This is a great suggestion and we will work towards adding this feature eventually. Unfortunately it's a bit far away on our roadmap.. We're currently revamping our design system (well, starting it from scratch basically) as well as frontend implementation, including CSS obviously. Until we have that, it won't be easy to add text resizing in some easy and, what's most important, maintainable way without doing some dirty hacks...

TL;DR: Definitely good idea but it has to wait a little since we're not where we want to be in order to implement that.

**Comment by maestromac** [comment]
Similar discussion https://github.com/thepracticaldev/dev.to/issues/2960

**Comment by cmgorton** [comment]
Thank you for opening this request.
Features tend to be subjective and might spur some debate. The Forem core team now uses an internal RFC ("request for comments") process to assess and prioritize new features. This process is intended to provide a consistent and standardized path for new changes to enter the Forem ecosystem. If you'd like to propose a new feature like this one, please visit [forem.dev](https://forem.dev/) to start a discussion around a new feature! 🙏 We have been working on more accessibility issues lately and I think it would be great if we could continue a discussion on this feature over on forem.dev to give it more visibility!
